components/wmtk_components/input/internal/mesh_with_tag_from_image.cpp

```diff
@@ -0,0 +1,80 @@
+#include "mesh_with_tag_from_image.hpp"
+
+#include <wmtk/utils/mesh_utils.hpp>
+
+#define STB_IMAGE_IMPLEMENTATION
+#include <stb_image.h>
+
+namespace wmtk::components::internal {
+
+Eigen::Matrix<long, -1, -1> tags_from_image(const std::filesystem::path& file)
+{
+    int width, height, channels;
+    unsigned char* img = stbi_load(file.string().c_str(), &width, &height, &channels, 0);
```

Comment:
who takes responsibility for closing this? i suspect there's a free at the end of this file

Reply:
#535